config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
```

Comment:
1. Open `.gitignore` file and add `config/database.yml` 
2. Run command: `git rm --cached config/database.yml`

Reply:
Done